### ISSUE TITLE
Server crashing when banning/pardoning

### ISSUE THREAD

**Issue by SnowBlitzz** [grandwazir/BanHammer#102]
Hey its snowblitzz and you probably already know my problem from the comments section. http://pastebin.com/NrD70gqi 
I am using mcpc, this version http://ci.md-5.net/job/MCPC-Plus-164/251/
BanHammer worked perfectly fine when I was using thr 2.3.1 version, UNTIL I updated it to 2.4.0, which then made every ban lag the server and most of the time, crash it. Even after I downgraded the version back to 2.3.1 and reset the config files and everything, problem still happens and I dont know why.

**Comment by grandwazir** [comment]
Thanks for reporting this issue. The fact that it a previous known good version is also having issues makes me think this may not be a bug in BanHammer itself. There were not database schema changes between 2.3.1 and 2.4.0 so that will also not be the issue. What it might be is a database connectivity issue. Does this only happen when banning players or do you also get delays when checking for bans and auditing?

**Comment by SnowBlitzz** [comment]
Checking for bans, auditing, ban recent, all that has no delay. Only happens when I ban, kick or pardon. Just these 3. How do I fix this? I also tried removing the database and restarted server so it would create a new one and see if it fixes, but nope :/

**Comment by grandwazir** [comment]
Hmm; thanks that is helpful information. Have you tried with no other plugins other than BanHammer?

**Comment by SnowBlitzz** [comment]
Yes, I tested that too. I don't understand why because it was working PERFECTLY fine, as in really perfectly before I updated to 2.4.0. Even after downgrading it back to 2.3.2 the problem still occurs. That's what gets me

**Comment by SnowBlitzz** [comment]
Any answer please? Sorry if I'm being annoying but this is the best ban plugin out there and nothing can replace it!! I'm really serious.

**Comment by grandwazir** [comment]
Sorry for the slow replies. I have been busy. I am not entirely sure what is going on here at all. Can you zip up a copy of a server that you have that replicates this issue and send it to me so I can attach a debugger? 

Also can you tell me what database you are using and how you have it setup. For example is it a local database or on a different server?

**Comment by SnowBlitzz** [comment]
Im not sure how to do that, the zipping part and stuff. Besides, the file is gonna be too big :/

I'm using the local database

**Comment by SnowBlitzz** [comment]
Currently, ban hammer is removed from my server and im just using essential bans. Wondering if 2.4.0 works with 1.6.4 because when I tried it, all these happened.

**Comment by grandwazir** [comment]
Should do. Give it a try and see if that fixes the issue.

**Comment by SnowBlitzz** [comment]
Same problem still occuring. Lags for a while when I /ban or /pardon and sometimes crashes
